assets/js/storage.js

```diff
@@ -1,6 +1,8 @@
 angular
 .module('storage', [])
 .factory('$storage', function() {
+  var likedVideos = getObj('likes');
```

Comment:
I don't know how angular works but my guess is that this value would get cached and wouldn't update as you favorited more videos unless when you favorite you update it